Client/Frontend/Browser/TabManager.swift

```diff
@@ -174,6 +176,10 @@ class TabManager : NSObject {
 
     // This method is duplicated to hide the flushToDisk option from consumers.
     func removeTab(tab: Browser) {
+        if tabsNetworkActivity[tab.webView!] == true {
+            decrementNetworkActivityCount()
+        }
+        tabsNetworkActivity.removeValueForKey(tab.webView!)
```

Comment:
I force unwrapped the tab's webView property, because I assumed that when closing a tab, the webView has already been created. It worked fine during my tests. Correct me if I'm wrong.